src/database/models/likes.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+export default (sequelize, DataTypes) => {
+  const Likes = sequelize.define('Likes', {
+    user_id: DataTypes.INTEGER,
+    accommodation_id: DataTypes.INTEGER,
+    status: DataTypes.BOOLEAN
+  }, {});
+  Likes.associate = (models) => {
+    // associations can be defined here
+  };
+  return Likes;
```

Comment:
you are missing the associations here, not sure if they are necessary but try and have them there, there is a r.ship between the users, accommodations and likes tables yeah?